config/routes.rb

```diff
@@ -8,6 +8,8 @@
 BACKUP_ROUTE_FORMAT = /.+\.(sql\.gz|tar\.gz|tgz)/i unless defined? BACKUP_ROUTE_FORMAT
 
 Discourse::Application.routes.draw do
+  def patch(*) end # Disable PATCH requests
```

Comment:
I think we need to improve the comment here to include the context of why Discourse applications do not support PATCH requests. This will help others to understand the rational of the code here many years later.

Reply:
@tgxworld Here I am, three years later, looking at the blame for context. I checked the rails topic, saw discourse, decided to look at the routes, noticed that line and landed here. 🙂